bip-musig2/vectors/sign_verify_vectors.json

```
@@ -92,7 +92,7 @@
                 "type": "value",
                 "message": "The signer's pubkey must be included in the list of pubkeys."
             },
-            "comment": "The signers pubkey is not in the list of pubkeys"
+            "comment": "The signers pubkey is not in the list of pubkeys. This test case is optional: it can be skipped by implementations that do not return an error."
```

```suggestion
            "comment": "The signers pubkey is not in the list of pubkeys. This test case is optional: it can be skipped for implementations that do not check that the signer's pubkey is included in the list of pubkeys."
```

I find this slightly clearer because "do not return an error" sounds a bit like "do not return errors at all".

---

done